### DISCUSSION TITLE
How to get a menu back you've hidden

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [freeplane/freeplane#875] [Help] [answered]
On my main install I already assigned a hotkey for hiding menus/toolbars. But on the alpha PortableApps version I'm using I didn't do that and I hid the menu forgetting I hadn't set one yet. Is there a way to get it back? If not, can someone just go ahead and make those two hiding controls come with a default hotkey?

**Comment by sgjohnson1981** [comment, marked as answer]
Nevermind. Found this post in the old forums on SourceForge.
https://sourceforge.net/p/freeplane/discussion/758437/thread/9f0a7b7d/

**Reply by EdoFro** [reply]
"Right click on a free area map and a menu appear that allow you to show it
again."
